module/VuFindConsole/src/VuFindConsole/Command/Util/SwitchDbHashCommand.php

```diff
@@ -63,55 +64,23 @@
 )]
 class SwitchDbHashCommand extends Command
 {
-    /**
-     * VuFind configuration.
-     *
-     * @var Config
-     */
-    protected $config;
-
-    /**
-     * User table gateway
-     *
-     * @var UserTable
-     */
-    protected $userTable;
-
-    /**
-     * UserCard table gateway
-     *
-     * @var UserCardTable
-     */
-    protected $userCardTable;
-
-    /**
-     * Config file path resolver
-     *
-     * @var PathResolver
-     */
-    protected $pathResolver;
-
     /**
      * Constructor
      *
-     * @param Config        $config        VuFind configuration
-     * @param UserTable     $userTable     User table gateway
-     * @param UserCardTable $userCardTable UserCard table gateway
-     * @param string|null   $name          The name of the command; passing null means
+     * @param Config                   $config          VuFind configuration
+     * @param UserTable                $userTable       User table gateway
+     * @param UserCardServiceInterface $userCardService UserCard database service
+     * @param string|null              $name            The name of the command; passing null means
      * it must be set in configure()
-     * @param PathResolver  $pathResolver  Config file path resolver
+     * @param ?PathResolver            $pathResolver    Config file path resolver
      */
     public function __construct(
-        Config $config,
-        UserTable $userTable,
-        UserCardTable $userCardTable,
+        protected Config $config,
+        protected UserTable $userTable,
+        protected UserCardServiceInterface $userCardService,
         $name = null,
```

Comment:
Could we add the type here? The parent constructor is typed.

Reply:
Yes, good catch!